pkg/util/shell/shell.go

```diff
@@ -74,7 +74,7 @@ func GetPrefixSuffixDelimiterForSet(userShell string, pathVar bool) (prefix, suf
 	switch userShell {
 	case "fish":
 		prefix = "set -gx "
-		suffix = "\";\n"
+		suffix = ";\n"
```

Comment:
I am not a fish user, but I am wondering why you are choosing to remove the closing quote over  adding a potentially "missing" opening quote? Does fish have different rules around quoting?